### PR TITLE
[WIP]: Add ha_cluster_exporter (1/3)

### PR DESCRIPTION
**Pull request by MalloZup**
# what does this PR?

- 1) it refactor states to be more modular and separated by domain (sap, susemanager)

- 2) add exporters  * check if it is ok the ha_cluster_exporter
This PR  is  wip but need to be completed see todos.

# todos:

- [ ] test  pr

# important note for reviewers:

@witekest @juliogonzalez  is this a problem if we don't add DEBIAN/RHEL packages how do you handle this?

### CODE REVIEW

File: prometheus-exporters-formula/metadata/form.yml [hunk 1]
```diff
@@ -65,3 +65,55 @@ postgres_exporter:
...
+# SAP/HA exporters
+ha_cluster_exporter:
+  $type: group
+  $help: Prometheus exporter for Pacemaker based HA clusters.
+
+  enabled:
+    $type: boolean
+    $default: False
+
+  address:
```
Comment (witekest) [inline prometheus-exporters-formula/metadata/form.yml:78]: This address should be passed to the exporter as an argument.

File: prometheus-exporters-formula/prometheus-exporters/suse-manager.sls [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    - context:
+        module: postgres
+    - user: root
+    - group: root
+    - mode: 644
+    - require:
+      - pkg: exporter_exporter
+    - watch_in:
+      - service: exporter_exporter
+{% endif %}
```
Comment (witekest) [inline prometheus-exporters-formula/prometheus-exporters/suse-manager.sls:189]: Empty line missing.

File: prometheus-exporters-formula/metadata/form.yml (hunk 1 above)
Comment (MalloZup) [reply prometheus-exporters-formula/metadata/form.yml:78]: just as understaing question why we don't do it for the other exporters?

**Comment by MalloZup** [comment]
closing this since we will install exporters via our sap formulas which we already have implemented for moniting

**Pull request closed without merging**